src/components/Calculator.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Button from './Button';
+import './Calculator.css';
+
+function Calculator() {
+  const buttons = ['AC', '+/-', '%', '÷', 7, 8, 9, 'x', 4, 5, 6, '-', 1, 2, 3, '+', 0, '.', '='];
+
+  return (
+    <div className="calculator">
+      <div className="result">0</div>
+      {buttons.map((buttonValue) => (
+        <Button key={buttonValue} value={buttonValue} />
+      ))}
+    </div>
+  );
+}
+
+export default Calculator;
```

Comment:
- [ ] [OPTIONAL] Kindly ensure that the  `Calculator.js` component should match the given design as stated in the project requirement.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="222" alt="Screenshot 2023-07-11 at 23 02 08" src="https://github.com/rudicarrilloypr/math-magicians/assets/58692590/0cbef3ef-f727-4cbc-b9c6-1ff986b9d17d"> | <img width="267" alt="Screenshot 2023-07-11 at 23 02 46" src="https://github.com/rudicarrilloypr/math-magicians/assets/58692590/bbd470d5-027e-460d-a13d-3ff9328bc23b">